number-guess/number_guess.rb

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env ruby
+require_relative "number_game"
+puts "Let's Play Guess the Number"
+print "Select difficulty (1,2,3):"
+
+level = gets.chomp
```

Comment:
```suggestion
level = gets.strip
```
so leading and trailing whitespace are also removed.